include/taskolib/Sequence.h

```diff
@@ -529,6 +540,18 @@ class Sequence
      */
     void set_label(gul14::string_view label);
 
+    /**
+     * Add one or more maintainers to the sequence. You are free to choose what ever you
+     * can use to identify the maintainer. You can also type more than one maintainer
+     * where they should be separated with comma or semicolon. The main maintainer should
+     * be enclose with square brackets:
```

Comment:
> The main maintainer should be enclose with square brackets

If we document such a "style guide" here, we should also enforce it. Otherwise it becomes meaningless – the users of Taskomat do not read the documentation of the library, after all. I propose to leave this part out.